src/ts/ssh/sshSession.ts

```diff
@@ -170,6 +170,8 @@ export class SshSession implements Disposable {
 	public trace: Trace = (level, eventId, msg, err) => {};
 
 	public constructor(public readonly config: SshSessionConfiguration, isClientSession?: boolean) {
+		this.isClientSession = isClientSession ?? false;
```

Comment:
nit: `this.isClientSession = !! isClientSession`

Reply:
Updated to make the property allow `undefined`, because it is possible to construct an `SshSession` that is neither specifically a client or a server (when not doing any encryption/auth).